pkgs/development/tools/minizinc/default.nix

```diff
@@ -1,17 +1,43 @@
-{ lib, stdenv, fetchFromGitHub, cmake, flex, bison }:
-stdenv.mkDerivation rec {
+{ lib, stdenv, fetchFromGitHub, callPackage, cmake, flex, bison, gecode, mpfr, cbc, zlib }:
+stdenv.mkDerivation (finalAttrs: {
   pname = "minizinc";
   version = "2.7.4";
 
-  nativeBuildInputs = [ cmake flex bison ];
+  nativeBuildInputs = [ cmake flex bison gecode mpfr cbc zlib ];
 
   src = fetchFromGitHub {
     owner = "MiniZinc";
     repo = "libminizinc";
-    rev = version;
+    rev = finalAttrs.version;
     sha256 = "sha256-Zq5gAwe9IQmknSDilFyHhSk5ZCQ8EfBOiM6Oef2WxYg=";
   };
 
+  postInstall = ''
+    mkdir -p $out/share/minizinc/solvers/
+    cat << EOF > $out/share/minizinc/solvers/gecode.msc
```

Comment:
I'm wondering if having this hard-coded here is a good idea. Wouldn't it be better to have an external JSON file?

And to replace strings inside you could use `jq`?

Reply:
I prefer the hard-coded string... but the next version of gecode is apparently going to provide its own msc file, so I guess we might as well figure out the `jq` incantation to fix the paths now.

Reply:
Fair enough.